Resources/Prototypes/_DV/Loadouts/Jobs/Science/roboticist.yml

```
@@ -37,3 +37,14 @@
   id: RoboticistPDA
   equipment:
     id: RoboticistPDA
+
+# Glasses
+- type: loadout
+  id: ClothingEyesHudDiagnostic
+  equipment:
+    eyes: ClothingEyesHudDiagnostic
+
+- type: loadout
+  id: ClothingEyesPrescriptionDiagHud
+  equipment:
+    eyes: ClothingEyesPrescriptionDiagHud
```

Add `ClothingEyesEyepatchHudDiag`.


```suggestion
- type: loadout
  id: ClothingEyesPrescriptionDiagHud
  equipment:
    eyes: ClothingEyesPrescriptionDiagHud
    
- type: loadout
  id: ClothingEyesEyepatchHudDiag
  equipment:
    eyes: ClothingEyesEyepatchHudDiag
```